### DISCUSSION TITLE
How to add a legend

### DISCUSSION THREAD

**Discussion by matbe81** [bozdoz/wp-plugin-leaflet-map#210] [General]
How to add a legend on the map as described here https://codepen.io/haakseth/pen/KQbjdO but using this nice plugins for wordpress?

**Comment by bozdoz** [comment]
Take a look at the FAQ, should be able to add a legend in the same way:

https://github.com/bozdoz/wp-plugin-leaflet-map?tab=readme-ov-file#how-can-i-add-another-leaflet-plugin
